pkg/referrerstore/plugin/skel/skel.go

```diff
@@ -117,7 +118,7 @@ func (c *pcontext) cmdListReferrers(cmdArgs *CmdArgs, pluginFunc ListReferrers)
 		}
 	}
 
-	result, err := pluginFunc(cmdArgs, cmdArgs.subjectRef, strings.Split(artifactType, ","), nextToken)
+	result, err := pluginFunc(cmdArgs, cmdArgs.subjectRef, strings.Split(artifactType, ","), nextToken, nil)
```

Comment:
nit: does golang support named parameter? for readability, would be good to know which parameter is passed as nil

Reply:
I don't think it does unfortunately. It doesn't support default value parameters either.